app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,30 @@
+class OrdersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @orders = Order.all
+  end
+
+  def show
+    @order = Order.find(params[:id])
+  end
+
+  def new
+    @order = Order.new(order_params)
+  end
+
+  def create
+    @order = current_user.orders.new(order_params)
+    if @order.save
```

Comment:
1. blank line after order initializing
2. layout else should be in the same column as if

Reply:
@VolodymyrZyblikevich